app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsScreenTest.kt

```diff
@@ -66,7 +66,7 @@ class StatisticsScreenTest {
         repository.apply {
             createTask("Title1", "Description1")
             createTask("Title2", "Description2").also {
-                completeTask(it.id)
+                completeTask(it)
```

Comment:
NIT: `also(::completeTask)`

Reply:
`also(::completeTask)` doesn't compile because `completeTask` is a suspending function and `also` expects a non-suspending function.

Reply:
![image](https://user-images.githubusercontent.com/873212/227920319-3c996272-078f-4a17-a3b1-e1c299b2a3b0.png)

Related: https://youtrack.jetbrains.com/issue/KT-36197/Accept-receiver-less-lambdas-where-a-receiver-is-provided